perl/lib/Sanger/CGP/Vaf/Data/AbstractVcf.pm

```diff
@@ -118,6 +156,15 @@ sub getBedIntervals {
     return shift->{'_b'};
 }
 
+sub _check_file_exists {
+  my ($self, $file) = @_;
+  die "set_input requires a value for $file" unless(defined $file);
+  die "Does not appear to be valid file: $file" if($file !~ m/\.bam$/ && $file !~ m/\.cram$/ && $file !~ m/\.vcf.gz$/);
```

Comment:
Not sure worth changing, but in future having the caller specify the type of file expected as a param would make this more robust.  As it stands passing in `vcf.gz` instead of `bam` would not give an error.

Reply:
as above